sovtokenfees/sovtokenfees/constants.py

```diff
@@ -18,4 +18,9 @@
 FEES_FIELD_NAME = 'fees'
 FEE_ALIAS_LENGTH = 128
 
-MAX_FEE_OUTPUTS = 1
\ No newline at end of file
+MAX_FEE_OUTPUTS = 1
+
+ACCEPTABLE_WRITE_TYPES = {FeesTransactions.SET_FEES.value, }
```

Comment:
This constants should be used in the FeesAuthNr initialization.

Reply:
ok